src/control/PlanarArmController.h

```diff
@@ -139,15 +139,25 @@ class PlanarArmController {
 	 *
 	 * @param currTime The current timestamp.
 	 * @param targetVel The target x velocity.
+	 * @param jointPos The joint angles of the arm.
```

Comment:
Kinda nitpicky, but can you specify that this is the *current* joint angles? Both here and the set_y_vel method